python/ray/rllib/ppo/env.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, name, batchsize, options):
 
     def reset(self):
         observations = [
-            self.preprocessor.transform(env.reset()) for env in self.envs]
+            self.preprocessor.transform(env.reset())[None] for env in self.envs]
```

Comment:
This works because for Mujoco environment,
```
obs = self.preprocessor.transform(env.reset())
np.vstack([obs]).shape == np.vstack([obs[None]]).shape
```